padrino-core/lib/padrino-core/cli/rake_tasks.rb

```diff
@@ -70,3 +62,11 @@ def env_migration_version
     list_app_routes(app, args) if app
   end
 end
+
+Dir["{lib/tasks/**,tasks/**,test,spec}/*.rake"].each do |file|
```

Comment:
These tasks need to be visible without environment manipulation for discovery reasons.

`rake -T` should _always_ show all tasks, independent of environment variables. Also, `test` and `spec` _activate_ the test environment.